CefSharp.Wpf/Internals/MouseTeleport.cs

```diff
@@ -0,0 +1,55 @@
+using CefSharp.Structs;
+
+namespace CefSharp.Wpf.Internals
+{
+    public class MouseTeleport
+    {
+        public Rect teleportingRect { get; private set; }
```

Comment:
`public` properties should start with a capital letter.

These should probably be private fields.

Reply:
I changed them to private fields.